### PR TITLE
Underconstruction

### PR DESCRIPTION
**Pull request by lucitemple**
This branch provides the UnderConstruction banner, which displays if process.env.NODE_ENV === 'production'.

We have not yet created unit tests for this, as we don't know how.

### CODE REVIEW

**Comment by lucitemple** [comment]
Just realised last commit failed and didn't push - this is not up to date to local changes. Will have to resolve issue before PR.

**Comment by hqtan** [comment]
Hi @lucitemple !

I've re-opened this PR, for a number of reasons:
- So we can continue to add fixes to this PR, to get the checks to pass :heavy_check_mark:
- so we (the reviewers) can provide you with constructive suggestions / feedback for your PR, have discussions on things pertinent to this PR; and then you can take that info to make more changes to the PR, and repeat the review - discuss - rewrite cycle, until the PR is in a state that both reviewers and contributors are agreeable with :smiley: 

A PR *does not* have to be 'perfect' on first submission. 
Reviewers should give feedback on things that should be further improved in the PR.

If you aren't finished with this PR, but would like some help/suggestions on how to proceed, you can mention it in the comments. You may also choose to prefix something like '[WIP]' to the PR's heading (e.g. '[WIP] Underconstruction') to indicate to others that this PR is a 'work-in-progress'.

So, in short:
Keep calm, and continue improving the PR :smiley: 
We'll get it merged eventually :smile: 

keep calm and carry on

**Comment by lucitemple** [comment]
Thanks @hqtan I think it is ready to be reviewed now :)

File: src/components/ConstructionBanner/ConstructionBanner.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import Container from 'react-bootstrap/Container';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (klubiniecki) [inline src/components/ConstructionBanner/ConstructionBanner.jsx]: We don't need this import, Bootstrap is already imported in `index.jsx`.

File: src/components/ConstructionBanner/ConstructionBanner.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+function ConstructionBanner() {
+  return (
+    <div>
```
Comment (klubiniecki) [inline src/components/ConstructionBanner/ConstructionBanner.jsx]: This `<div>` doesn't do anything, it can be removed.

File: src/components/ConstructionBanner/ConstructionBanner.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      <Container fluid className="p1">
+        Learn more. Demand better. SeeChange.
+        <br />
+        {' '}
+        Coming soon.
+        <div className="p2">
+          <p />
```
Comment (klubiniecki) [inline src/components/ConstructionBanner/ConstructionBanner.jsx]: What is this `<p />` tag doing?

File: src/components/ConstructionBanner/ConstructionBanner.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import Container from 'react-bootstrap/Container';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function ConstructionBanner() {
+  return (
+    <div>
+      <Container fluid className="p1">
```
Comment (klubiniecki) [inline src/components/ConstructionBanner/ConstructionBanner.jsx]: Why did you decide to name those classes p1 and p2, and what styles are attached to them?

File: src/components/HomePage/HomePage.jsx [hunk 5]
```diff
@@ -1,23 +1,30 @@
...
-        you want to see around you.
-        <div className="p2">
-          <p />
-          Find how your elected officals have voted in relation with climate
-          change and
-          <u>make your voice heard.</u>
-        </div>
-      </Container>
-
+      {process.env.NODE_ENV === 'production' ? (
```
Comment (klubiniecki) [inline src/components/HomePage/HomePage.jsx]: The purpose of banner is to replace the whole text, as per design in Trello card. In your code, it just replaces the first part of the home page. It should be taken outside of the return statement and returned instead of the whole home page code.

**Review by klubiniecki — CHANGES_REQUESTED** [review]

File: src/components/HomePage/HomePage.jsx (hunk 5 above)
Comment (lucitemple) [reply src/components/HomePage/HomePage.jsx]: Oh! Sorry, we misunderstood that, we thought it was just the banner that was going to be switched. Will correct :)

**Comment by lucitemple** [comment]
@klubiniecki Thanks for taking the time to review and providing feedback and change requests. We have made the changes and it is ready to be reviewed again.

**Review by klubiniecki — APPROVED** [review]
Great work, thanks 😊
